snuba/datasets/plans/translators.py

```diff
@@ -0,0 +1,31 @@
+import copy
+
+from abc import ABC, abstractmethod
+
+from snuba.query.physical import Query as PhysicalQuery
+from snuba.query.logical import Query as LogicalQuery
+
+
+class QueryTranslator(ABC):
```

Comment:
This is really just a placeholder at this point so that we are mindful that there is an explicitly called translation phase in query plan building. I expect the implementations of this class to evolve a lot after we nail down the design of the query translation